src/mixins/actionText.js

```diff
@@ -61,6 +74,17 @@ export default {
 	],
 
 	computed: {
+		/**
+		 * TODO: drop on the 8.0.0 major, see title/name prop
+		 */
+		nameTitleFallback() {
+			if (this.name === null && this.title) {
```

Comment:
The `title` prop was not required, so the new `name` prop should not be required either. We only warn if there is a `title` but no `name` (it's of course valid to do so, but we want to draw attention to this change, so some false warnings might be ok).